plugin/completion/lib_complete.py

```diff
@@ -388,7 +396,7 @@ def _parse_completions(complete_results, excluded):
                     continue
                 hint += chunk.spelling
                 if chunk.isKindTypedText():
-                    trigger = chunk.spelling
+                    trigger += chunk.spelling
```

Comment:
This is strange. It changes the behavior for c++ and c too. Is this intended? Why?